read-the-docs/source/JIRA.rst

```diff
@@ -25,6 +26,18 @@ Methods
 -  `add_watcher() <#add_watcher>`__
 -  `add_attachment() <#add_attachment>`__
 
+get_ticket_content()
+--------------------
+
+``get_ticket_content(self, ticket_id=None)``
+
+Queries the JIRA API to get ticket_content using ticket_id.
+
+.. code:: python
+
+    t = ticket.get_ticket_content('Ticket_ID')
```

Comment:
Can you use '<ticket_id>' instead of 'Ticket_ID' here?

Also, can you add a note that the ticket_content will be of the form specified in the JIRA API at the following location: https://docs.atlassian.com/software/jira/docs/api/REST/7.6.1/#api/2/issue-getIssue

Reply:
Also, if you could add a quick note to this section about how the user accesses the ticket_content after running get_ticket_content(), i.e. returned_ticket_content = t.ticket_content, or something along those lines.

Reply:
I do not understand precisely the documentation conventions here, so I changed it based on my intuition. So let me know if something else needs some corrections.